src/data/finset/lattice.lean

```diff
@@ -1108,7 +1120,7 @@ begin
   simp_rw [val_to_finset],
 end
 
-lemma sup_eq_bUnion {α β} [decidable_eq β] (s : finset α) (t : α → finset β) :
+@[simp] lemma sup_eq_bUnion {α β} [decidable_eq β] (s : finset α) (t : α → finset β) :
```

Comment:
I'm not sure we should be touching this lemma - I suspect you added `simp` by accident instead of adding it to a lemma I mentioned above?

@YaelDillies, thoughts on this simp lemma?

Reply:
I'm sorry, let me un'simp it.
(Obviously, I need to be tutored on how to use the conversation tab of github. Or maybe one just learns by practicing it.)

Reply:
Are you aware you can commit suggestions directly from github, rather than copying them over by hand?